<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Входные данные</string>
    </property>
    <widget class="QPushButton" name="addPurchaseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Отчество:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="firstNameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="countOfPhysicsEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="middleNameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберите учебники:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lastNameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="physicsCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Физика</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mathCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Математика</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="infoCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Информатика</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="countOfMathEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="countOfInfoEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QPushButton" name="clearListButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>671</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>671</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Физика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Математика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Информатика</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="queryPushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить запрос</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetTableButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>671</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
